### ISSUE TITLE
Refactor|Rename leaves the cursor in limbo.

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#2077]
Running 2.0.5rc

I'm a keyboard kinda guy, so I hit the <kbd>right mouse button</kbd> key, followed by <kbd>b</kbd>, <kbd>r</kbd>, <kbd>r</kbd> to get to the Rubberduck Rename function. I type in my new name, then hit enter. 

The item is appropriately renamed, but the cursor/insertion point is nowhere to be found. Using the arrow keys doesn't make it visible, nor do they cause either the IDE's nor RD's cursor location (RxCy) to change.

Hitting <kbd>Alt</kbd><kbd>tab</kbd>to change code modules will give me my cursor back, as will reaching for the mouse and clicking in the editor window, however, reaching for the mouse defeats the purpose of using the keyboard in the first place, and hitting <kbd>Alt</kbd><kbd>Tab</kbd> then <kbd>Alt</kbd><kbd>Shift</kbd><kbd>Tab</kbd> seems like it shouldn't be necessary.

**Comment by Hosch250** [comment]
Try CTRL-Shift-R next time.

**Comment by retailcoder** [comment]
Hmm... the IDE's focus/cursor position would be in the last module processed by RD in the course of the refactoring I think - there's no way of knowing where that is, but IMO there's a way to give the original module (where focus was before the refactoring was triggered) back the focus once we're done.

If that's the case, then other refactorings and tools would do the same thing, like removing/reordering parameters of a procedure with callers in another module, or applying an inspection's quick-fix across multiple modules (e.g. "fix all occurrences in project"). Can you confirm that is the case?

**Comment by Hosch250** [comment]
It isn't just when there are references in multiple modules.

**Comment by daFreeMan** [comment]
Renaming just happened to be what I was doing when I noticed the problem, so that's what I focused in on.

@retailcoder - I can confirm that reordering parameters also leaves the cursor in limbo. Tested on a procedure defined in A, called from B.

@Hosch250 - that would be great if I hadn't mapped Ctrl-Shift-R to bring up the CE (since RD is still stealing hotkeys and I need Ctrl-R and....  waitaminute hotkey thievery has been fixed!  Yay!!
